### ISSUE TITLE
Unexpected token )

### ISSUE THREAD

**Issue by shanselman** [felixrieseberg/npm-windows-upgrade#81]
```
Windows PowerShell
Copyright (C) 2016 Microsoft Corporation. All rights reserved.

Loading personal and system profiles took 624ms.
PS C:\WINDOWS\system32> Set-ExecutionPolicy Unrestricted -Scope CurrentUser -Force
PS C:\WINDOWS\system32> npm install --global --production npm-windows-upgrade
>>
C:\Users\Scott\AppData\Roaming\npm\npm-windows-upgrade -> C:\Users\Scott\AppData\Roaming\npm\node_modules\npm-windows-upgrade\bin\npm-windows-upgrade.js
npm-windows-upgrade@3.0.1 C:\Users\Scott\AppData\Roaming\npm\node_modules\npm-windows-upgrade
├── cli-spinner@0.2.5
├── commander@2.9.0 (graceful-readlink@1.0.1)
├── chalk@1.1.3 (escape-string-regexp@1.0.5, supports-color@2.0.0, ansi-styles@2.2.1, strip-ansi@3.0.1, has-ansi@2.0.0)
├── promise@7.1.1 (asap@2.0.4)
├── inquirer@1.0.3 (ansi-escapes@1.4.0, mute-stream@0.0.6, through@2.3.8, strip-ansi@3.0.1, cli-width@2.1.0, figures@1.7.0, pinkie-promise@2.0.1, run-async@2.2.0, string-width@1.0
.1, cli-cursor@1.0.2, rx@4.1.0, lodash@4.13.1)
├── prompt@1.0.0 (revalidator@0.1.8, read@1.0.7, colors@1.1.2, pkginfo@0.4.0, winston@2.1.1, utile@0.3.0)
└── regenerator-runtime-only@0.8.38 (promise@6.1.0, es6-symbol@2.0.1)
PS C:\WINDOWS\system32> npm-windows-upgrade
C:\Users\Scott\AppData\Roaming\npm\node_modules\npm-windows-upgrade\bin\npm-windows-upgrade.js:28
  .then(() => upgrader.ensureInternet())
         ^
SyntaxError: Unexpected token )
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
```

**Comment by shanselman** [comment]
Looks like old version of Node. Failed early. I updated and now it works. FYI.

**Comment by felixrieseberg** [comment]
Thanks a bunch for the issue :bow:, there was some new syntax that snuck through the transpiler - it's now fixed in 3.0.2.
